CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# Changelog
+
+v1.11.0-RC2 / 2020-06-11
```

Comment:
@shubham14bajpai -- does the unversioned files need to be moved as part of this PR to the specific release folder.